FE/frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { Button } from 'antd';
+import styled from 'styled-components';
+import LoginForm from '../components/login/LoginForm';
+import RegisterForm from '../components/login/RegisterForm';
+import {
+  StyledPage,
+  StyledFormWrapper,
+} from '../components/styled/StyledLoginPage';
+import { useAuthStateValue } from '../context/AuthContext';
+import {
+  ILoginRequest,
+  IRegisterRequest,
+} from '../service/authorization/authorization.props';
+
+const Login: React.FC = () => {
+  const [isLogin, setIsLogin] = useState<boolean>(true);
```

Comment:
You do not need to explicitly set a type here. TypeScript will infer a boolean type based on your default value.

Reply:
Updated.